File: api/CES.Business/Models/ExhibitSearchResultModel.cs
using System.Collections.Generic;

namespace CES.Business.Models
{
    // One row per non-deleted exhibit (file), carrying its parent submission's
    // context for display. Exhibit-centric — a flat list, not submission-grouped.
    public class ExhibitSearchResultModel
    {
        public SubmissionFile File { get; set; } = new();
        public int SubmissionId { get; set; }
        public DateTime? SubmissionDate { get; set; }
        public string? AppearanceDateTime { get; set; }
        public string Location { get; set; } = string.Empty;
        public string Room { get; set; } = string.Empty;
        public List<string> FileNumbers { get; set; } = new();
        public string? AccusedName { get; set; }
    }
}

File: web/src/models/ExhibitSearchResultModel.ts
import type { SubmissionFile } from './SubmissionReviewModel';

// One row per exhibit (mirrors the backend ExhibitSearchResultModel). Superset of
// ExhibitList.vue's PriorFileEntry ({ file, submissionDate?, fileNumbers[] }), so
// each result maps directly onto an entry.
export interface ExhibitSearchResultModel {
  file: SubmissionFile;
  submissionId: number;
  submissionDate?: string;
  appearanceDateTime?: string;
  location: string;
  room: string;
  fileNumbers: string[];
  accusedName?: string;
}

export interface ExhibitSearchFilter {
  fileNumberText?: string;
  accusedName?: string;
  appearanceDateFrom?: string;
  appearanceDateTo?: string;
}
